### PR TITLE
Povelikin Rostislav lab1

### PR DESCRIPTION
**Pull request by PovelikinRostislav**

### CODE REVIEW

**Comment by PovelikinRostislav** [comment]
@nik709
@Maxim-Doronin

**Comment by PovelikinRostislav** [comment]
@Lyzzerg

**Comment by Lyzzerg** [comment]
https://www.youtube.com/watch?v=NP_msZDHbbg

**Comment by nik709** [comment]
👍

**Comment by Lyzzerg** [comment]
:+1:

File: modules/complex-number/test/test_povelikin_rostislav_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Povelikin Rostislav
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Povelikin_Rostislav_ComplexNumberTest, can_create_not_zero) {
```
Comment (Maxim-Doronin) [inline modules/complex-number/test/test_povelikin_rostislav_complex_number.cpp]: This test is a copy of `Can_Create_With_Real_And_Imaginary` test in `test_complex_number.cpp` file

File: modules/complex-number/test/test_povelikin_rostislav_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    // Act
+    ComplexNumber z(re, im);
+
+    // Assert
+    EXPECT_EQ(re, z.getRe());
+    EXPECT_EQ(im, z.getIm());
+}
+
+TEST(Povelikin_Rostislav_ComplexNumberTest, setters_are_correct) {
```
Comment (Maxim-Doronin) [inline modules/complex-number/test/test_povelikin_rostislav_complex_number.cpp]: And this one looks like `Can_Set_Real` and `Can_Set_Imaginary`

**Pull request closed without merging**